conseil-api/src/main/resources/metadata/quorum.mainnet.conf

```diff
@@ -0,0 +1,7 @@
+// Since currently both networks (JP Morgan Quorum and Ethereum) have the same data entites,
+// we can use the same metadata configuration for both
+{
+  include "ethereum.mainnet.conf"
+
+  visible: true
```

Comment:
I'm just wondering, what if we moved the visibility flag outside the metadata file into the main "including file"?

That would centralize the switches, and allow to easily check the situation at a glance.

What do you think? @ajuszczak @molowny ?

Reply:
Do you mean, something like that: 
```
ethereum {
    visible: true
    networks {
      mainnet {
        include "metadata/ethereum.mainnet.conf"
        visible: true
      }
    }
  }
```
in the `metadata.conf`? If so, then I like it a bit more. I mean, at least you see that this specific network is visible. Which is more important to see quickly than which fields we are exposing from specific entities in the network. I like the idea.

Reply:
Yes, exactly what I meant

Reply:
Let me change that and we will see :)

Reply:
I'm not sure if this would mean that we have to read the metadata differently than now...

When we include, does a sub-object gets created within the network object, or are the included-object fields "merged" with the including?

To make clear: does the `visible` attribute in the included metadata object already appear as an attribute of the `mainnet` network? Or is it nested in another object?
Personally I can't see no other way to think about it, since the included object is not assigned to any field of the including object, so a form of merging of attributes must happen

Reply:
They will be merged (with latest appearance of the same field being the one set at the end). Final file will looks exactly the same. I'm not planning to change reading implementation. Just configuration file.

Reply:
So:
```
include "ethereum.mainnet.conf"
```
and 
```
{
   include "ethereum.mainnet.conf"
}
```
technically is the same. I had to add `{ }`, because I wanted to śet visibility field. Since we do not need it anymore, I can simply have single include file there.